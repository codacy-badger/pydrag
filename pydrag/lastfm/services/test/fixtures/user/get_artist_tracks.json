{
    "version": 1,
    "interactions": [
        {
            "request": {
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&artist=Trivium&format=json&method=user.getArtistTracks&page=1&user=rj",
                "body": null,
                "headers": {
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Accept": [
                        "*/*"
                    ],
                    "Connection": [
                        "keep-alive"
                    ]
                }
            },
            "response": {
                "status": {
                    "code": 200,
                    "message": "OK"
                },
                "headers": {
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:59 GMT"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Content-Length": [
                        "125"
                    ]
                },
                "body": {
                    "string": "{\"artisttracks\":{\"track\":[],\"@attr\":{\"user\":\"RJ\",\"artist\":\"Trivium\",\"page\":\"1\",\"perPage\":\"50\",\"totalPages\":\"0\",\"total\":\"0\"}}}"
                }
            }
        }
    ]
}
