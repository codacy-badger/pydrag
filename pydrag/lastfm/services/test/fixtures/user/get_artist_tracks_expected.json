{
    "@attr": {
        "page": 1,
        "user": "RJ",
        "total": 0,
        "artist": "Trivium",
        "perPage": 50,
        "totalPages": 0
    },
    "track": []
}
